cfep-11.md

```diff
@@ -0,0 +1,28 @@
+
+<table>
+<tr><td> Title </td><td> Simple staged-recipes Automation </td>
```

Comment:
IMO, the title doesn't reflect what is being proposed.

Reply:
Sure! Happy to change!

Reply:
I changed the title @isuruf. LMK if you think this is better!